src/components/data_comps/docn/nuopc/docn_comp_mod.F90

```diff
@@ -595,7 +595,7 @@ subroutine docn_comp_run(x2o, o2x, &
     integer           :: nu     ! unit number
     character(len=18) :: date_str
     character(len=CS) :: fldname
-
+    character(len=CS) :: local_case_name
```

Comment:
CS isn't long enough for a case name. I'd use at least CL (as you do for the mct version).